specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json

```diff
@@ -350,6 +375,19 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "enum": [
+              "stats"
+            ],
+            "x-ms-enum": {
+              "name": "GetShareExpand"
```

Comment:
kind reminder: Do you want to model as String here?

Reply:
I have no strong preference on modelAsString. Will set it to false to match StorageAccountExpand, which is modeled as enum.